R/epipredict-data.R

```diff
@@ -1,12 +1,14 @@
-#' Subset of JHU daily state cases and deaths
+#' JHU daily COVID-19 cases and deaths rates from all states
 #'
-#' This data source of confirmed COVID-19 cases and deaths
-#' is based on reports made available by the Center for
-#' Systems Science and Engineering at Johns Hopkins University.
-#' This example data ranges from Dec 31, 2020 to Dec 31, 2021,
-#' and includes all states.
+#' This data source of confirmed COVID-19 cases and deaths is based on reports
+#' made available by the Center for Systems Science and Engineering at Johns
+#' Hopkins University, as downloaded from the CMU Delphi COVIDcast Epidata
+#' API. This example data ranges from Dec 31, 2020 to Dec 31, 2021, and
+#' includes all states. It is used in the {epiprocess} correlation vignette.
 #'
-#' @format A tibble with 20,496 rows and 4 variables:
+#' @format An object of class \code{epi_df} (inherits from \code{tbl_df}, \code{tbl}, \code{data.frame}) with 37576 rows and 4 columns.
```

Comment:
```suggestion
#' @format An `epi_df` (object of class `c("epi_df", "tbl_df", "tbl", "data.frame")`) with 37576 rows and 4 columns.
```
musing: I'm not sure if `epi_df` should be a link like ``[`epi_df`][epiprocess::epi_df]``, not a link, or something like ``[`epiprocess::epi_df`]``.  The ``[`epi_df`][epiprocess::epi_df]`` way seems better than just ``[`epi_df`]``, as it generates errors if we try documenting without epiprocess installed, and if we get something pre-documented, package checks are clearer this way as well (complaining about not having epiprocess for crossref check rather than a missing topic).  But there's not good feedback in my editor at least when trying this link if epiprocess isn't installed (just some generic message about not finding the link, not that I'm missing epiprocess), and I don't know if it's enough for CRAN.

musing: The possibilities of users not having epiprocess or tibble installed make documenting what they'll actually see a bit frustrating.  Guess we could have a wordy explanation here

Reply:
Switched all similar `@format` cases to using "[\`epiproces...\`]", and added a warning about df types when datasets are loaded without `epiprocess` being installed.